homeassistant/components/azure_devops/__init__.py

```diff
@@ -87,7 +87,7 @@ def device_info(self) -> DeviceInfo:
             "identifiers": {
                 (
                     DOMAIN,
-                    self.organization,
+                    self.organization,  # type: ignore
```

Comment:
I'm confused why this is type ignored? The parent class specifies it as a string its constructor?

Reply:
because without it: mypy gives me:
homeassistant/components/azure_devops/__init__.py:88: error: Argument 1 to <set> has incompatible type "Tuple[str, str, str]"; expected "Tuple[str, str]"  [arg-type]

I could probably put the ignore in line 88, if that is better.

Reply:
Oh right, this is actually "broken" (as in works, but not correct).

Reply:
Done.

Reply:
yup, an interesting job...make it right, and convert existing devices....the conversion could stay for a couple of releases and then be removed.